src/App.css

```diff
@@ -5,65 +5,7 @@
 }
```

Comment:
![image](https://user-images.githubusercontent.com/93039359/235873862-522023a0-8f62-4604-ba66-dac5e3e70496.png)

**[OPTIONAL]**

- [ ] It would be nice if you remove the first title because you already have a title after Table of contents 🤔 so there is not point of having two titles so kindly remove the first one which is highlighted in the image above ⬆️